src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/læremidler/domain/BeregningsresultatLæremidler.kt

```diff
@@ -14,19 +15,11 @@ data class BeregningsresultatForMåned(
 )
 
 data class Beregningsgrunnlag(
-    val måned: YearMonth,
-    val studienivå: Studienivå,
-    val studieprosent: Int,
-    val sats: Int,
-)
-
-data class BeregningPeriode(
     override val fom: LocalDate,
     override val tom: LocalDate,
+    val utbetalingsMåned: YearMonth,
```

Comment:
Hvis det er `s` der burde det kanskje være litet m 😬 eller `utbetalingMåned` selv om det ikke er korrekt norsk
```suggestion
    val utbetalingsmåned: YearMonth,
```